covid/transform.py

```diff
@@ -914,6 +954,25 @@ def transform_covidtracking_data(covidtracking_df):
                 ),
             ] = negative_streak_series.values
 
+        # Calculate policy vs. trend charts data.
+        # Calculate raw cases per million.
+        covidtracking_df.loc[(state,), POLICY_VS_TREND_RAW_CASES_PER_MILLION] = (
+            covidtracking_df.loc[(state,), NEW_CASES_FIELD].values / state_population
+        ) * ONE_MILLION
+
+        # Calculate 3DCS cases per million.
+        covidtracking_df.loc[(state,), POLICY_VS_TREND_3DCS_CASES_PER_MILLION] = (
+            covidtracking_df.loc[(state,), NEW_CASES_3DCS_FIELD].values
+            / state_population
+        ) * ONE_MILLION
+
+        # Calculate positivity 3DCS.
+        covidtracking_df.loc[
+            (state,), POLICY_VS_TREND_3DCS_POSITIVITY
```

Comment:
Do we need to copy the `FRACTION_POSITIVE_NEW_TESTS_3DCS_FIELD` column into `POLICY_VS_TREND_3DCS_POSITIVITY`, or can we just use the former?